src/rust/cryptography-x509/src/extensions.rs

```diff
@@ -291,6 +291,24 @@ pub struct NamingAuthority<'a> {
     pub text: Option<DisplayText<'a>>,
 }
 
+type SequenceOfDisplayTexts<'a> = common::Asn1ReadableOrWritable<
+    asn1::SequenceOf<'a, DisplayText<'a>>,
+    asn1::SequenceOfWriter<'a, DisplayText<'a>, Vec<DisplayText<'a>>>,
+>;
+
+type SequenceOfObjectIdentifiers<'a> = common::Asn1ReadableOrWritable<
+    asn1::SequenceOf<'a, asn1::ObjectIdentifier>,
+    asn1::SequenceOfWriter<'a, asn1::ObjectIdentifier, Vec<asn1::ObjectIdentifier>>,
+>;
+
+pub struct ProfessionInfo<'a> {
+    pub naming_authority: Option<NamingAuthority<'a>>,
+    pub profession_items: SequenceOfDisplayTexts<'a>,
+    pub profession_oids: Option<SequenceOfObjectIdentifiers<'a>>,
+    pub registration_number: Option<asn1::PrintableString<'a>>,
+    pub add_profession_info: Option<asn1::Tlv<'a>>,
```

Comment:
This should be an octet string (`&'a [u8]`), not a TLV.

Reply:
I thought at first this should be it, [even tried that in the POC](https://github.com/pyca/cryptography/commit/d6a7b1563b827ff30eb655d14ae4cfff32153abc#diff-44c30c40fd401c71ad99b759debcbc52001793695a65fe349f93a20dd37e9baf)), but found no other examples for using bytes types in fields, so refrained from using it. Great that my initial thought wasn't that bad :-)